torch/_dynamo/variables/tensor.py

```diff
@@ -582,18 +582,8 @@ def call_method(
                 if result:
                     return result
             except TypeError as e:
-                unimplemented(f"unhandled args for {name}: {e}")
+                unimplemented(f"unhandled args for {name}: {e}")        
 
-        from .builder import wrap_fx_proxy
-
-        return wrap_fx_proxy(
```

Comment:
Can you call `unimplemented(f"tensor method {name} NYI")` at the bottom of this function?

Reply:
Hi @mlazos, called the unimplemented method at the end of the function ([a8668ee](https://github.com/pytorch/pytorch/pull/140746/commits/a8668eefef9983e5c95c21565f839123b5010ea0)). Kindly check and review.

Thanks!